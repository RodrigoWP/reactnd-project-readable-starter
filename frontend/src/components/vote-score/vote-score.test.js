'use strict'

import React from 'react'
import { shallow } from 'enzyme'
import { VoteScore } from '.'

it('Should render correctly', () => {
  const component = <VoteScore icon='' />

  expect(shallow(component)).toMatchSnapshot()
})
