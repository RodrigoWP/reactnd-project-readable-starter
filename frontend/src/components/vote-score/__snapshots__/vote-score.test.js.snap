// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<ScoreIcon
  count={0}
  icon=""
/>
`;
